modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -267,8 +268,15 @@ class Instrument_Manager extends \NDB_Menu_Filter
     {
         $factory  = \NDB_Factory::singleton();
         $filename = $this->path . "project/instruments/$instname.linst";
-        $fp       = fopen($filename, "r");
-        $db       = \Database::singleton();
+        if (!is_readable($filename)) {
+            if (file_exists($filename)) {
+                return '? (Insufficient read permissions)';
+            } else {
+                return '? (File not found in filesystem)';
+            }
+        }
```

Comment:
This change is very misleading. The "?" is for PHP instruments where it can't determine the status. "Insufficient read permission" should be an error, not an unknown status, and "File not found" is expected for non-LINST instruments. This makes it sound like an error, but it's the expected result for NDB_BVL_Instrument instruments.